api/app/src/main/kotlin/packit/clients/GithubUserClient.kt

```diff
@@ -38,14 +37,25 @@ class GithubUserClient(private val config: AppConfig, private val githubBuilder:
         return UserPrincipal.create(user, mutableMapOf())
     }
 
-    fun checkGithubOrgMembership()
+    fun checkGithubMembership()
```

Comment:
should we probably be  returning something here? like true or something? seems a bit weird to let method run and relying on throwing error?

Reply:
That's kind of the idea though, since this is being called from both the OAuth2UserService (browser login) and GithubAPILoginService (API login) - they're both delegating the throwing of the exception to the user client if there's a problem with the user authentication. I'll add an annotation to the method though, to make it more explicit what the usage is.